_posts/blog/2016-05-12-ehcache3_jsr107_spring.adoc

```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: "Ehcache 3, JSR-107 and Spring 4.2"
+headline: "Ehcache 3 has been released!"
```

Comment:
I believe the headline should be a longer description of what your post is about.